### PR TITLE
Feat/support moving cursor by j and k key.

### PR DESCRIPTION
**Pull request by khanghoang**
Hi, I want to support moving the cursor by `j` and `k` key so that you don't need to move your hand too far. Those keys are `Vim` standard movement.

### CODE REVIEW

**Comment by khanghoang** [comment]
@vadimdemedes please help me to review this one. Thanks.

File: test.js [hunk 1]
```diff
@@ -60,6 +60,7 @@ test('list - custom indicator', t => {
 
 	const CustomIndicator = () => 'X ';
 
+        // eslint-disable-next-line  react/jsx-no-bind
```
Comment (vadimdemedes) [inline test.js]: These eslint comments aren't aligned with the code and I don't think they are necessary. There's no `bind()` on the next line.

**Comment by vadimdemedes** [comment]
Nice idea, thanks for the PR! One minor detail and this should be good to go.

**Comment by khanghoang** [comment]
@vadimdemedes updated :)

**Comment by khanghoang** [comment]
hmmm, for some reason CI build failed because of these errors

```
  test.js:63:49
  ✖   63:49  JSX props should not use arrow functions                       react/jsx-no-bind
  ✖   81:49  JSX props should not use arrow functions                       react/jsx-no-bind
```

**Comment by vadimdemedes** [comment]
Hm, weird, I'll take a look.

**Comment by khanghoang** [comment]
I just checked the code. These lint errors were there before.

**Comment by vadimdemedes** [comment]
Ok, merging then, thanks for the PR ;)

**Pull request merged**